requirements_test.txt

```diff
@@ -1,2 +1,3 @@
+coverage>=4.0,<4.4
```

Comment:
Can you add a TODO comment so we can find the reason behind the change without digging commit logs in the future?